_v2_source/index.html

```diff
@@ -0,0 +1,162 @@
+
+        .splitflap {
+            margin: 0 auto;
+            -webkit-perspective-origin: top;
+            -moz-perspective-origin: top;
+            -ms-perspective-origin: top;
+            perspective-origin: top;
+            -webkit-perspective: 900px;
+            -moz-perspective: 900px;
+            -ms-perspective: 900px;
+            perspective: 900px;
+        }
+
+        #dashboard {
+            display: block;
+            width: 100%;
+            margin: 0 auto;
+            overflow: hidden;
+        }
+
+        #firstSection {
+            float: left;
+        }
+
+        #secondSection {
+            margin-left: 5px;
+            float: left;
+        }
+
+        #thirdSection {
+            margin-left: 5px;
+            float: left;
+        }
+    </style>
+    <script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
+    <script src="js/guests.js"></script>
+    <script src="js/ordering.js"></script>
+    <script src="js/jquery.splitflap.js"></script>
+</head>
+<body>
+    <div class="heading"></div> <br />
+    <div id="dashboard">
+        <div id="firstSection"></div>
+        <div id="secondSection"></div>
+        <div id="thirdSection"></div>
+    </div>
+
+    <script>
+        (function ($) {
+            $(document).ready(function () {
+                var guests = getGuests();
+                var ordering = getOrdering(guests.length);
+
+                initHeading($);
+                initDashboard($, guests);
+
+                window.setTimeout(function () {
+                    animateDashboard($, ordering, guests.length, 0);
+                }, 2000);
+
+                window.setTimeout(function () {
+                    refreshDashboardRandomly($, guests, null);
+                }, 2000 * (guests.length + 10));
+
+            });
+        })(jQuery);
+
+        function initHeading($) {
+            var headingZoomRatio = 0.5;
+            $('.heading')
+                .splitFlap({
+                    text: 'Zsoka es Peti eskuvoi jarata',
+                    charWidth: 50 * headingZoomRatio,
+                    charHeight: 100 * headingZoomRatio,
+                    imageSize: (2500 * headingZoomRatio) + 'px ' + (100 * headingZoomRatio) + 'px'
+                });
+        }
+
+        function initDashboard($, guests) {
+            var rowZoomRatio = 0.3;
+            var nameColumnWidth = 19;
+            var tableColumnWidth = 7;
+            var rowsInSection = 18;
+            for (var i = 0; i < guests.length; i++) {
+                var id = 'rows_' + i;
+                var sectionId = i < rowsInSection ? 'firstSection' : (i < 2 * rowsInSection ? 'secondSection' : 'thirdSection');
+                var guest = guests[i];
+                var newDomElement = '<div id=' + id + '>' + guest.name.padEnd(nameColumnWidth) + ' ' + guest.table.padEnd(tableColumnWidth) + '</div>';
+
+                $('#' + sectionId).append(newDomElement);
+                $('#' + id).splitFlap({
```

Comment:
no need to create and use that many ids for sections you can build on the fly the elements and insert children into them as well - makes easier to maintain the DOM structure.